pyvista/plotting/charts.py

```diff
@@ -1376,20 +1373,15 @@ def show(
         ----------
         off_screen : bool, optional
             Plots off screen when ``True``.  Helpful for saving screenshots
-            without a window popping up.  Defaults to active theme setting in
-            :attr:`pyvista.global_theme.full_screen
-            <pyvista.themes.DefaultTheme.full_screen>`.
+            without a window popping up. Defaults to active theme setting.
 
         full_screen : bool, optional
             Opens window in full screen.  When enabled, ignores
-            ``window_size``.  Defaults to active theme setting in
-            :attr:`pyvista.global_theme.full_screen
-            <pyvista.themes.DefaultTheme.full_screen>`.
+            ``window_size``. Defaults to active theme setting.
 
-        screenshot : str or bool, optional
+        screenshot : str or bool, default: False
             Saves screenshot to file when enabled.  See:
             :func:`Plotter.screenshot() <pyvista.Plotter.screenshot>`.
-            Default ``False``.
 
             When ``True``, takes screenshot and returns ``numpy`` array of
             image.
```

Comment:
I can't add a comment on lines 1391-1392, but I would like the description of `window_size` to be consistent with the descriptions of `off_screen` and `full_screen` (either as it was before these changes with linking, or shortened to "Defaults to active theme setting" without linking).

Reply:
Discovered online editing through github.dev today and gave it a shot to make this change, pretty neat :)